pkg/customresource/actionset.yaml

```diff
@@ -11,49 +11,78 @@ spec:
     singular: actionset
   scope: Namespaced
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
-    schema:
-      openAPIV3Schema:
-        properties:
-          apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          properties:
```

Comment:
Do we need changes in this file @r4rajat ?

Reply:
Yes @viveksinghggits, as we're adding RepositoryServer field in actionset, so we've to modify it's customresource definition